kong/consumer_group_test.go

```diff
@@ -348,6 +348,15 @@ func TestConsumerGroupGetEndpointPostGW39(t *testing.T) {
 	require.NoError(err)
 	assert.NotNil(consumerGroupFromKong)
 	assert.Equal(consumerGroupFromKong.ConsumerGroup.ID, createdConsumerGroup.ID)
-	// Consumers are not listed in a get operation
+	// Consumers are listed
+	assert.NotNil(consumerGroupFromKong.Consumers)
+	assert.Len(consumerGroupFromKong.Consumers, 1)
+
+	// Check GetWithConsumers
+	consumerGroupFromKong, err = client.ConsumerGroups.GetWithNoConsumers(defaultCtx, createdConsumerGroup.ID)
+	require.NoError(err)
+	assert.NotNil(consumerGroupFromKong)
+	assert.Equal(consumerGroupFromKong.ConsumerGroup.ID, createdConsumerGroup.ID)
+	// Consumers are not listed
 	assert.Nil(consumerGroupFromKong.Consumers)
```

Comment:
For readability, I'd suggest putting assertions regarding `GetWithNoConsumers` and `Get` methods into separate test cases. Also, instead of comments, it might be better to pass a message to the assertion. This way, we get to have the same context from the reader's perspective and additionally get a nice message if the test fails. Like so:
```suggestion
	t.Run("GetWithNoConsumers", func(t *testing.T) {
		consumerGroupFromKong, err := client.ConsumerGroups.GetWithNoConsumers(defaultCtx, createdConsumerGroup.ID)
		require.NoError(err)
		assert.NotNil(consumerGroupFromKong)
		assert.Equal(consumerGroupFromKong.ConsumerGroup.ID, createdConsumerGroup.ID)
		assert.Nil(consumerGroupFromKong.Consumers, "consumer should not be listed")
	})
```